### PR TITLE
[Qt] add defaultConfirmTarget constant to sendcoinsdialog

### PR DESCRIPTION
**Pull request by Diapolo**
- replaces some hard-coded values for the default confirmation target

### CODE REVIEW

**Comment by jonasschnelli** [comment]
utACK @cfields trivial branch

**Comment by Diapolo** [comment]
Same here, even if it is a small change, I've been told by @theuni and @laanwj that these don't belong into trivial.

**Comment by sipa** [comment]
utACK.

**Comment by fanquake** [comment]
utACK

File: src/qt/sendcoinsdialog.cpp [hunk 1]
```diff
@@ -628,7 +628,7 @@ void SendCoinsDialog::updateSmartFeeLabel()
     if(!model || !model->getOptionsModel())
         return;
 
-    int nBlocksToConfirm = (int)25 - (int)std::max(0, std::min(24, ui->sliderSmartFee->value()));
+    int nBlocksToConfirm = (int)defaultConfirmTarget - (int)std::max(ui->sliderSmartFee->minimum(), std::min(ui->sliderSmartFee->maximum(), ui->sliderSmartFee->value()));
```
Comment (laanwj) [inline src/qt/sendcoinsdialog.cpp]: > (int)std::max(ui->sliderSmartFee->minimum(), std::min(ui->sliderSmartFee->maximum(), ui->sliderSmartFee->value()

Does the slider guarantee this value range for its value, making this redundant?

Comment (Diapolo) [inline src/qt/sendcoinsdialog.cpp]: [quote]
When setting this property, the minimum is adjusted if necessary to ensure that the range remains valid. Also the slider's current value is adjusted to be within the new range.
[/quote]

Seems like this IS the case...

Comment (Diapolo) [inline src/qt/sendcoinsdialog.cpp]: So this would become ``nTxConfirmTarget = defaultConfirmTarget - ui->sliderSmartFee->value();``, right?

Comment (Diapolo) [inline src/qt/sendcoinsdialog.cpp]: @laanwj Updated to include your suggestion...

**Comment by Diapolo** [comment]
@laanwj Can you re-check please?

**Pull request merged**